CI/merge_hashes.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python3
```

Comment:
will this be used by the CI?

Reply:
Not really, it's just convenient for updating I think, that's why I wanted to ship it with the repo. The hashes are FOR the CI though, so that's why it's there. Do you have another place in mind?

Reply:
no I think that makes perfect sense